holoviews/tests/teststreams.py

```diff
@@ -893,7 +893,9 @@ def transform_function(cls, stream_values, constants):
         v = sum([val["v"] for val in stream_values if val["v"]])
         return dict(v=v + constants['base'])
 
-Val = Stream.define("Val", v=0.0)
+
+class Val(Stream):
+    v = param.Number(constant=True)
```

Comment:
I'm fine with this either way, but shouldn't the `Stream.define` version be equivalent?

Reply:
The reason I changed that was that I was cloning streams using:

```python
cloned_stream = type(stream)(**stream.contents)
```

And this wasn't working for streams build with `Stream.define`. Is there a better way to clone streams? Maybe we should look at adding a `Stream.clone` method.

Reply:
Actually, this does work. Not sure what I was running into during development.  In https://github.com/holoviz/holoviews/pull/4551/commits/711f082e81b13e17824e054c55ac994b71047156 I reverted some of these to use `Stream.define` so that there is a mix of both in the tests.